jdaviz/app.py

```diff
@@ -197,37 +197,25 @@ def __init__(self, configuration=None, *args, **kwargs):
     @property
     def hub(self):
         """
-        Reference to the stored application handler hub instance.
-
-        Returns
-        -------
-        `glue.core.Hub`
-            The internal ``Hub`` instance for the application.
+        Reference to the stored application handler `~glue.core.hub.Hub` instance
+        for the application.
```

Comment:
Note: `@property` do not really need a "Returns" section. They must return something by definition.